data/rulesets.js

```diff
@@ -60,6 +60,13 @@ exports.BattleFormats = {
 		effectType: 'ValidatorRule',
 		name: 'Pokemon',
 		desc: ["The foundational rules for any and all formats based on in-game mechanics (everything but Custom Game)"],
+		onStart: function (battle) {
+			let ruleTable = this.getRuleTable(this.getFormat(battle.format));
```

Comment:
`this.getFormat(battle.format)`

is equivalent to `this.getFormat()`

Reply:
`this === battle`, right?

Reply:
Apparently yes, `this === battle`.Feels like I should have checked that first. I was not aware of the former either.